themes/grass/layouts/download/list.html

```diff
@@ -41,8 +41,8 @@ <h2 class="section-title">Download your free and open source copy</h2>
                   <div class="col-4 text-center"><object type="image/svg+xml" data="{{.Site.BaseURL}}/images/logos/grasslogo.svg" class="grasslogo"></object></div>
                   <div class="col-8">
 		    <h3 class="mt-20">GRASS GIS source code</h3>
-		    <p>The GRASS GIS source code is available on <a href="https://github.com/OSGeo/grass" target="_blank"><i class="fa fa-github"></i> Github</a></p>
-		    <p class="command d-none d-lg-block"><a href="https://github.com/OSGeo/grass"> $ git clone https://github.com/OSGeo/grass </a></p>
+		    <p>The GRASS GIS source code is available on <a href="https://github.com/OSGeo/grass" target="_blank"><i class="fa fa-github"></i> Github</a>, see all <a href="https://github.com/OSGeo/grass/releases">releases</a></p>
+            <p class="command d-none d-lg-block"><a href="https://github.com/OSGeo/grass"> $ git clone https://github.com/OSGeo/grass </a></p>
```

Comment:
something broke indentation, could be tabs instead spaces or missing spaces

Reply:
There were indeed some tabs there, I replaced them now by spaces. Does it fix it for you?

Reply:
not really, you changed also the other lines, probably you need to maintain the tabs, I saw the problem here with Github changes viewer